pkg/landscaper/installations/imports/validation_test.go

```diff
@@ -141,16 +141,16 @@ var _ = Describe("Validation", func() {
 		defer ctx.Done()
 		inInstA, err := installations.CreateInternalInstallation(ctx, op, fakeInstallations["test1/a"])
 		Expect(err).ToNot(HaveOccurred())
-		inInstA.Info.Status.Phase = lsv1alpha1.ComponentPhaseProgressing
-		Expect(fakeClient.Update(ctx, inInstA.Info)).To(Succeed())
+		inInstA.GetInfo().Status.Phase = lsv1alpha1.ComponentPhaseProgressing
+		Expect(fakeClient.Update(ctx, inInstA.GetInfo())).To(Succeed())
 
 		inInstB, err := installations.CreateInternalInstallation(ctx, op, fakeInstallations["test1/b"])
 		Expect(err).ToNot(HaveOccurred())
-		inInstB.Info.Status.Phase = lsv1alpha1.ComponentPhaseSucceeded
+		inInstB.GetInfo().Status.Phase = lsv1alpha1.ComponentPhaseSucceeded
```

Comment:
As we moved to the struct approach I think we do not need that getter indirection anymore.
So we could just leave most of the code as it is
```suggestion
		inInstB.Info.Status.Phase = lsv1alpha1.ComponentPhaseSucceeded
```